superbuild/projects_modules/zlib.cmake

```diff
@@ -0,0 +1,49 @@
+function(zlib_project)
```

Comment:
With ZLIB, It was necessary on windows to do the install step and to set  INSTALL_** variables in order to have the headers in the build directory.

Reply:
the install command is always done, unless specified. I added the install path in a new commit.